xbmc/windowing/wayland/WinSystemWaylandEGLContextGLES.cpp

```diff
@@ -44,8 +44,10 @@ bool CWinSystemWaylandEGLContextGLES::InitWindowSystem()
 
   CLinuxRendererGLES::Register();
 
+#ifndef TARGET_WEBOS
```

Comment:
ifdeffing a registration method seems wrong. The whole point of the registration factory is to determine if it can be used.

Currently CDVDVideoCodecDRMPRIME doesn't do any checks but it should be updated to check if ffmpeg supports a hwcontext with AV_PIX_FMT_DRM_PRIME. If it doesn't then that means ffmpeg wasnt built with libdrm support.

Reply:
I've pushed an attempt at checking for DRM presence in ffmpeg. It correctly detects the the unavailability of DRM on webOS. However, I don't have a DRM device to test if it also correctly detects the presence of a DRM device.